pkg/network/http/compile.go

```diff
@@ -0,0 +1,26 @@
+// +build linux_bpf
+
+package http
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/ebpf/bytecode/runtime"
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+)
+
+//go:generate go run $PWD/pkg/ebpf/include_headers.go $PWD/pkg/network/ebpf/c/runtime/http.c $PWD/pkg/ebpf/bytecode/build/runtime/http.c $PWD/pkg/ebpf/c $PWD/pkg/network/ebpf/c/runtime $PWD/pkg/network/ebpf/c
```

Comment:
We had to replace `$PWD` since it doesn't work in Go 1.17 (see #9139). Could you fix up this file too?